users/dierkes/preprocessing/segment_cutting.py

```diff
@@ -15,42 +18,91 @@ class CutAndStitchSpeechSegmentsFromCorpusJob(Job):
 
 
     """
-    def __init__(self, bliss_corpus_file, target_length=20, n_workers=4, file_extension='wav'):
+
+    def __init__(
+        self,
+        bliss_corpus_file,
+        min_length=10,
```

Comment:
I think so far I am the only person using the job. I don't like to excluding `min_length` from the hash as it does as a big impact on the result. I will set `target_length` back to the default value of 20 again.

Reply:
`__sis_hash_exclude__` is not for excluding something from the hash but to extent existing jobs with new parameters without changing its hash, see [here](https://github.com/rwth-i6/sisyphus/blob/f875d5f921a54cce09b3eb5353ae75c629ab08df/sisyphus/job.py#L161). But if you're the only one, then this doesn't matter.

Reply:
`min_length` is now excluded from the hash when being 0

Reply:
I don't think this makes sense. You should exclude it when it is 10 if you want to keep your hashes. But actually the behavior changes with this, so it's actually even desirable to have different hashes. In that case you can remove the `__sis_hash_exclude__`.

Reply:
We want to keep the hashes of the old version of the Job, don't we? In the old version the parameter `min_length` did not exist and the value that should generate the same behavior is 0. However I only believe the behavior is the same. As the logic is a little bit different now it might not be exactly the same. Given that so far only I used the Job it is maybe best to just remove `__sis_hash_exclude__` anyways.

Reply:
Ah, I think I get what you mean now. We would have to set the default value of `min_length` to 0 and then ignore it for the value 0 so that the job to really behaves like its old version and not breaking the hashes. So your are right, we should remove `__sis_hash_exclude__`